src/cmd/add.go

```diff
@@ -26,14 +26,20 @@ var addCmd = &cobra.Command{
 		if err != nil {
 			panic(err)
 		}
-		if !template.HasTemplatesDir(appDir) {
-			fmt.Println("no templates found\n\nPlease add templates to the \".exosphere\" folder of your code base.")
-			os.Exit(1)
+		hasTemplates, err := template.HasTemplatesDir(appDir)
+		if err != nil {
+			panic(err)
```

Comment:
I see. Is it a programming error that requires seeing the stack track (in order to change something in the program so that we can avoid this error in the future)? In that case let's use a panic. 

Or is that a runtime error where all we can do is show the user the error message and abort? Then we should use `log.fatalf`.